src/track/beatmap.h

```diff
@@ -82,7 +79,7 @@ class BeatMap final : public Beats {
     BeatMap(const BeatMap& other);
 
     // For internal use only.
```

Comment:
Same here, adding such a comment to a private method is confusing. Unrelated, but let's clean this up now.